packages/internals/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -823,4 +823,42 @@ describe('getGenerators', () => {
       )
     }
   })
+
+  test('pass if no model(s) found but allow-no-models flag is passed - sqlite', async () => {
+    expect.assertions(1)
+
+    const aliases = {
+      'predefined-generator': {
```

Comment:
I'm assuming you meant to change this one too?
```suggestion
      'prisma-client-js': {
```